build-n-release-w-docker.sh

```diff
@@ -23,17 +23,20 @@ export MSYS2_ARG_CONV_EXCL="*"
 
 mkdir -p dist
 
-docker build --pull -t doppler-ui-system-source --target test .
+docker build --pull -t doppler-ui-system-source .
```

Comment:
I think that `--target test` is better because you are not taking advantage of the build